src/lib/components/column-profile/CollapsibleTableHeader.svelte

```diff
@@ -67,10 +67,7 @@
 
   const clickEntityNameHandler = () => {
     dispatch("select");
-    if (
-      entityType == EntityType.Table ||
-      (entityType == EntityType.Model && active)
-    ) {
+    if (entityType == EntityType.Model && active) {
```

Comment:
To address my comment about expanding the profiling info on the second click, what about:

```javascript
if ((entityType == EntityType.Table || entityType == EntityType.Model) && active)
```